lib/ansible/modules/cloud/ovirt/ovirt_storage_domains.py

```diff
@@ -284,6 +307,13 @@ def build_entity(self):
             name=self._module.params['name'],
             description=self._module.params['description'],
             comment=self._module.params['comment'],
+            wipe_after_delete=self._module.params['wipe_after_delete'],
+            discard_after_delete=self._module.params['discard_after_delete'],
+            backup=self._module.params['backup'],
+            critical_space_action_blocker=self._module.params['critical_space_action_blocker']
+            if self._module.params['critical_space_action_blocker'] else None,
+            warning_low_space_indicator=self._module.params['warning_low_space']
+            if self._module.params['warning_low_space'] else None,
```

Comment:
Let's remove the `if` and set default `None`.

Reply:
done